src/app/cli/src/graphql.ml

```diff
@@ -1001,7 +1018,9 @@ module Subscriptions = struct
 
   let new_block =
     subscription_field "newBlock"
-      ~doc:"Fires on a new block created by a proposer with a public key KEY"
+      ~doc:
+        "Fires on a new block involving the specified public key - either by \
+         producing this block or by having a transaction included in it"
```

Comment:
"Event that triggers when a new block is created that either contains a transaction with the specified public key, or was produced by it"